### PR TITLE
revive the doc for building without nix

### PR DESCRIPTION
**Pull request by chenyan-dfinity**

### CODE REVIEW

**Comment by chenyan-dfinity** [comment]
@kentosugama

**Comment by nomeata** [comment]
Before merging, did someone test if this still works reliably?

File: Building.md [hunk 1]
```diff
@@ -97,6 +97,34 @@ Make a PR off of that branch and merge it using a _normal merge_ (not
 squash merge) once CI passes. It will eventually be imported into this
 repo by a scheduled `niv-updater-action`.
 
+## Development without nix-shell
+
+You can get a development environment without having to use `nix-shell`
+(although installing all required tools without nix is out of scope).
+
+ * Use your system’s package manager to install `ocaml` (4.10) and
```
Comment (ggreif) [inline Building.md:105]:
```suggestion
 * Use your system’s package manager to install `ocaml` (4.12) and
```

File: Building.md [hunk 2]
```diff
@@ -97,6 +97,34 @@ Make a PR off of that branch and merge it using a _normal merge_ (not
...
+   [`opam`](https://opam.ocaml.org/doc/Install.html)
```
Comment (ggreif) [inline Building.md:106]: what about `dune` (2.0)?

**Comment by luc-blaeser** [comment]
Just doing some repo cleanup for better focussing on active work:
Closing PR as it seems inactive for a longer time. Please re-open if it is still relevant and after it has been updated with latest master changes.

**Pull request closed without merging**